### ISSUE TITLE
Barcode Plate B addition to Traction LIMS for PacBio

### ISSUE THREAD

**Issue by Stan1520** [sanger/asset_audits#701]
**User story**

As a User, I want to.....

**Who are the primary contacts for this story**

- Craig Corton - chc@sanger.ac.uk
- James Watts - jw24@sanger.ac.uk

**Who is the nominated tester for UAT**

- Craig Corton - chc@sanger.ac.uk
- James Watts - jw24@sanger.ac.uk

**Acceptance criteria** 
To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**

This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
@Stan1520 I think this story is for traction service? Also the fasta file is not attached.

**Comment by Stan1520** [comment]
Hi Stephen. Yes it is for Traction. I am also getting a different format for the file from the team. Once I have it, I’ll attach it to the story.

Cheers,
Stanley
________________________________

**Comment by Stan1520** [comment]
@stevieing Shall I forward you the mail with the file format? It seems to be the only format available. You might be able to open and upload here. For some reason, I am unable to view and upload it directly.

**Comment by stevieing** [comment]
> @stevieing Shall I forward you the mail with the file format? It seems to be the only format available. You might be able to open and upload here. For some reason, I am unable to view and upload it directly.

Yes please. I can turn it into a text file and upload it. I will move this issue to the correct repository.
